stubs/django/db/models/deletion.pyi

```diff
@@ -0,0 +1,25 @@
+from typing import Collection
+
+from django.db.models.base import Model
```

Comment:
The `Model` errors are because we directly [defined `Model` in the `__init__.py` file](https://github.com/facebook/pyre-check/blob/5a85dd491427553b1799a2789ced4e76252e7fdd/stubs/django/db/models/__init__.pyi#L44), rather than importing it from `base.py` into the init file; moving it is the right solution, but I'm happy to do that in a follow up commit and you can just change your imports to `from django.db.modelsimport Model` instead.